test/parallel/test-async-hooks-async-await.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const async_hooks = require('async_hooks');
+const assert = require('assert');
+
+const asyncIds = [];
+async_hooks.createHook({
+  init: (asyncId, type, triggerAsyncId) => {
+    asyncIds.push([triggerAsyncId, asyncId]);
+  }
+}).enable();
+
+async function main() {
+  console.log('hello');
```

Comment:
if the `console.log()` statements are not required, then let's not have them. If they are required for the test, please add a comment so folks don't remove them